README.md

```diff
@@ -58,49 +58,43 @@ Usage
 Example setup
 -------------
 
-To build manylinux, macOS, and Windows wheels on Travis CI and upload them to PyPI whenever you tag a version, you could use this `.travis.yml`:
+To build manylinux, macOS, and Windows wheels on Github and make them availble for download after the job runs, you could use this `.github/workflows/wheels.yml`:
 
 ```yaml
-language: python
+name: Build
+
+on: [push, pull_request]
 
 jobs:
-  include:
-    # perform a linux build
-    - services: docker
-    # and a mac build
-    - os: osx
-      language: shell
-    # and a windows build
-    - os: windows
-      language: shell
-      before_install:
-        - choco install python --version 3.8.0
-        - export PATH="/c/Python38:/c/Python38/Scripts:$PATH"
-        # make sure it's on PATH as 'python3'
-        - ln -s /c/Python38/python.exe /c/Python38/python3.exe
-
-env:
-  global:
-    - TWINE_USERNAME=__token__
-    # Note: TWINE_PASSWORD is set to a PyPI API token in Travis settings
-
-install:
-  - python3 -m pip install cibuildwheel==1.7.1
-
-script:
-  # build the wheels, put them into './wheelhouse'
-  - python3 -m cibuildwheel --output-dir wheelhouse
-
-after_success:
-  # if the release was tagged, upload them to PyPI
-  - |
-    if [[ $TRAVIS_TAG ]]; then
-      python3 -m pip install twine
-      python3 -m twine upload wheelhouse/*.whl
-    fi
+  build_wheels:
+    name: Build wheels on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-18.04, windows-latest, macos-latest]
```

Comment:
why `ubuntu-18.04` instead of `ubuntu-latest`?

Reply:
I have no idea, that's what's in the example. Mayne because `ubuntu-latest` generates a warning that it will change to `ubuntu-20.04` soon, but that example predated that warning.

I think for a 2020 example, skipping Python 2 might be useful in the example? `CIBW_SKIP: "cp27-*" # Skip building Python 2.7 on all platforms` for example?

Reply:
> I think for a 2020 example, skipping Python 2 might be useful in the example? `CIBW_SKIP: "cp27-*" # Skip building Python 2.7 on all platforms` for example?

This does not skip pypy27. But I agree that short example may skip python 2.7 Also this allow to remove "Install Visual C++ for Python 2.7" step.

Reply:
In general, I prefer to lock to specific versions when writing CI configs, so in that case, it should be`os: [ubuntu-20.04, windows-2019, macos-10.15]`. I think previously it was only possible to lock the ubuntu image, but now you can lock windows and macos too.

I also like the idea of adding a single CIBW setting.

Reply:
```suggestion
        os: [ubuntu-20.04, windows-2019, macOS-10.15]
```

Assuming we want 10.15 instead of 11.0, as it's still listed as preview?

Reply:
Yeah that's what I had assumed, because `latest` pointed to 10.15